src/checks/lowercase_key.rs

```diff
@@ -0,0 +1,51 @@
+use crate::checks::{Check, Warning};
+use crate::LineEntry;
+
+pub(crate) struct LowercaseKeyChecker {}
```

Comment:
This struct should have the field `warning: Warning`.
Example: https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/leading_space.rs#L5

Reply:
sorry, It seems kinda weird to me create warning with message template: 
```rust
pub(crate) struct LowercaseKeyChecker {
    warning: Warning
}

// like this
impl Default for LowercaseKeyChecker {
    fn default() -> Self {
        Self {
            warning: Warning::new("The {} key should be in uppercase"),
        }
    }
}

// then i should extract message template from warning
//...
Some(Warning {
    message: format!(self.warning.clone(), key) // btw it doesnt work: format! receives string literal on the first place
})
```
can you explain, why we should create struct with warning, only for storing message template, why we can't store only message? :thinking:

Reply:
Yes, you're right. I missed that. What do you think about this solution?
```rust
pub(crate) struct LowercaseKeyChecker {
    template: String
}

impl Default for LowercaseKeyChecker {
    fn default() -> Self {
        Self {
            warning: String::from("The {} key should be in uppercase"),
        }
    }
}

Some(Warning {
    message: Some(Warning { message: self.template.replace("{}", key) })
})
```

Reply:
seems ok, i'll take it